bindings/core/src/method/wallet.rs

```diff
@@ -194,7 +195,13 @@ pub enum WalletMethod {
     /// Prepares to transition an implicit account to an account.
     /// Expected response: [`PreparedTransaction`](crate::Response::PreparedTransaction)
     #[serde(rename_all = "camelCase")]
-    PrepareImplicitAccountTransition { output_id: OutputId },
+    PrepareImplicitAccountTransition {
+        output_id: OutputId,
+        #[serde(default)]
+        public_key: Option<PublicKey>,
```

Comment:
Since this is not our type, how does this serialise ? prefixed hex ?

Reply:
HMMMM I doubt it. probably as bytes. What do we want?

Reply:
prefixed_hex I'd say, but maybe this is already the case, have to check

Reply:
done